### ISSUE TITLE
Reopen access point when connection lost

### ISSUE THREAD

**Issue by iball8** [resin-io/resin-wifi-connect#165]
Reopen the Wifi Connect AP when the wifi connection is lost. This feature would be useful if someone decided to use a different wifi connection in the event they change locations or the previous wifi connection permanently fails.

**Comment by majorz** [comment]
Indeed, right now the only way for doing this is relaunching the application (e.g. when the application container is restarted when running on top of resinOS using approach as in our new start script). This should work for the case where the location is changed and the device is rebooted, but not for others.

Making improvements towards better permanent connectivity is on our roadmap for sure, but there are not any concrete details yet that I can provide.

For now an one way would be to launch a separate background process which periodically checks for the wireless connectivity state and launches the application if needed. I will try to find time to getting something like that implemented and added to our scripts folder.

**Comment by iball8** [comment]
Zahari,

Restarting the application should work fine in my case. We will just need to explain network reconnection in our user manual.

I’ll also look into doing this and update if any progress is made. Appreciate the response!

Thanks,
Ivan

Sent from my iPhone

On Jan 29, 2018, at 4:13 AM, Zahari Petkov <notifications@github.com<mailto:notifications@github.com>> wrote:


Indeed, right now the only way for doing this is relaunching the application (e.g. when the application container is restarted when running on top of resinOS using approach as in our new start script). This should work for the case where the location is changed and the device is rebooted, but not for others.

Making improvements towards better permanent connectivity is on our roadmap for sure, but there are not any concrete details yet that I can provide.

For now an one way would be to launch a separate background process which periodically checks for the wireless connectivity state and launches the application if needed. I will try to find time to getting something like that implemented and added to our scripts folder.
